src/index.js

```diff
@@ -21,32 +21,50 @@ const __filename = fileURLToPath(import.meta.url);
 const __dirname = dirname(__filename);
 
 if (process.env.BASIC_AUTH_ENABLED) {
-  app.use((req, res, next) => {      
+  app.use((req, res, next) => {
     const b64auth = (req.headers.authorization || '').split(' ')[1] || '';
-    const [login, password] = Buffer.from(b64auth, 'base64').toString().split(':');
-  
-    if (login === process.env.BASIC_AUTH_LOGIN && password === process.env.BASIC_AUTH_PASSWORD) {
+    const [login, password] = Buffer.from(b64auth, 'base64')
+      .toString()
+      .split(':');
```

Comment:
For the future, if you want to use a dependency in your local environment, it would also be helpful to add it to the `package.json`, so that everyone who is on the project can keeping the style consistent.

Reply:
Thanks. Updated formatting dependencies [here](https://github.com/together-support/togethernet/pull/57/commits/13bf968b9b709719a4695b19b73775647e9b0f3f).